src/hotspot/os/posix/os_posix.cpp

```diff
@@ -1024,47 +1064,6 @@ char* os::Posix::describe_pthread_attr(char* buf, size_t buflen, const pthread_a
   return buf;
 }
 
-char* os::Posix::realpath(const char* filename, char* outbuf, size_t outbuflen) {
```

Comment:
I'm looking at this from the GitHub UI so I might be missing something, but why was this moved up?

Reply:
I'd assume because it is now os::foo rather than os::Posix::foo - though such a move is not necessary.